code/modules/reagents/newchem/drinks.dm

```diff
@@ -231,5 +233,6 @@
 	result = "synthignon"
 	required_reagents = list("synthanol" = 1, "wine" = 1)
 	result_amount = 2
+	mix_message = "The chemicals mix to create a fine, red substance."
```

Comment:
Ew, ew, already used enough, meh, what, and why. Is winey even a word? It sounds like complaining. I like it being 'fine' though. Should it really be changed?